command/common_output.go

```diff
@@ -1,14 +1,21 @@
 package command
 
+import "sync"
+
 type commonOutputFormatter struct {
+	sync.Mutex
 	errorOutput   error
 	commandOutput CommandResult
 }
 
 func (c *commonOutputFormatter) SetError(err error) {
+	c.Lock()
```

Comment:
I don't see any getError locking? Did you decide to make lock in place?

Reply:
Basically, this is not needed anymore. Removed.